bot/cogs/utils.py

```diff
@@ -40,6 +40,14 @@
 Namespaces are one honking great idea -- let's do more of those!
 """
 
+PEP0_TITLE = "Index of Python Enhancement Proposals (PEPs)"
+PEP0_INFO = {
+    "Status": "Active",
+    "Created": "13-Jul-2000",
+    "Type": "Informational"
+}
+PEP0_LINK = "https://www.python.org/dev/peps/"
```

Comment:
Adding constants feels like clutter when these are unlikely to change and aren't used in more than one place. I suggest you just use the values directly in code.